epochStart/bootstrap/disabled/disabledChainStorer.go

```diff
@@ -0,0 +1,100 @@
+package disabled
+
+import (
+	"errors"
+
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+// ChainStorer is a mock implementation of the ChianStorer interface
+type ChainStorer struct {
+	AddStorerCalled func(key dataRetriever.UnitType, s storage.Storer)
```

Comment:
this is a stub. not really needed. you could use an inmemory storer if you want to throw it away.

Reply:
Agree, but this is a storer chain. I would leave it like this and change only the storer below to a in-memory one